lib/python/rose/macro.py

```diff
@@ -50,6 +50,8 @@
 ERROR_LOAD_CHOSEN_META_PATH = "Could not find metadata for {0}, using {1}\n"
 ERROR_LOAD_META_PATH = "Could not find metadata for {0}"
 ERROR_LOAD_CONF_META_NODE = "Error: could not find meta flag"
+ERROR_MACRO_CASE_MISMATCH = "Error: Namelist option mismatch; \n {0} does not\
+ match {1}, please only use lowercase."
```

Comment:
Lowercase is just for namelists - we might insist that environment variables be uppercase or something in the future.